spec/jobs/create_derivatives_job_spec.rb

```diff
@@ -19,15 +19,13 @@
       allow_any_instance_of(GenericFile).to receive(:mime_type).and_return(mime_type)
       @generic_file.save!
     end
-    context 'with a video (.avi) file', unless: $in_travis do
+    context 'with a video (.avi) file' do
       let(:mime_type) { 'video/avi' }
       let(:file_name) { 'countdown.avi' }
 
-      it 'lacks a thumbnail' do
+      it 'generates a thumbnail on job run' do
+        skip 'hydra-derivatives require proprietary custom codecs that are not included in ffmpeg, skip this test'
```

Comment:
skipped this test as it has a crazy infrasture requirement...you need a very specially built ffmpeg for this to even work...